src/base/static/components/input-form/index.js

```diff
@@ -319,66 +319,76 @@ class InputForm extends Component {
       includePrivate: this.props.hasAdminAbilities(this.props.datasetSlug),
     });
 
-    if (placeResponse) {
-      Util.log("USER", "new-place", "successfully-add-place");
-
-      // Save attachments.
-      if (this.attachments.length) {
-        await Promise.all(
-          this.attachments.map(async attachment => {
-            const attachmentResponse = await mapseedApiClient.attachments.create(
-              placeResponse.url,
-              attachment,
-            );
-
-            if (attachmentResponse) {
-              placeResponse.attachments.push(attachmentResponse);
-              Util.log("USER", "dataset", "successfully-add-attachment");
-            } else {
-              alert("Oh dear. It looks like an attachment didn't save.");
-              Util.log("USER", "place", "fail-to-add-attachment");
-            }
-          }),
-        );
-      }
+    if (!placeResponse) {
+      alert("Oh dear. It looks like that didn't save. Please try again.");
+      Util.log("USER", "place", "fail-to-create-place");
+      return;
+    }
+    if (placeResponse.isOffline) {
+      alert(
+        "No internet connection detected. Your submission may not be successful until you are back online.",
+      );
+      Util.log("USER", "place", "submitted-offline-place");
+      this.props.router.navigate("/", { trigger: true });
```

Comment:
@goldpbear I added this line to address your PR feedback. Thanks for the tip :+1: